ui/app/styles/core/alert-banner.scss

```diff
@@ -1,3 +1,6 @@
+.has-margin-top {
```

Comment:
I think there's a class that already exists with this styling, `.has-top-margin-s`

Reply:
You're right, in the helper. I was searching for has-margin and it was has-top... 🤦🏻‍♀️